lib/Github/Api/Repo.php

```diff
@@ -25,7 +25,8 @@
 class Repo extends AbstractApi
 {
     /**
-     * Search repositories by keyword:
+     * Search repositories by keyword:.
```

Comment:
@GrahamCampbell: in this case, you just need to remove the colon before the period.